README.md

```diff
@@ -64,7 +64,7 @@ The app can then be removed with:
 
 Curtail uses a number of open source projects to work properly:
 
-- [GTK 3](https://www.gtk.org)
+- [GTK 4](https://www.gtk.org)
```

Comment:
As this a Circle app, it is expected that it uses libadwaita, this grants support for dark mode.